groups/storage/main.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_version = ">= 1.3.0, < 1.4.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = "eu-west-2"
```

Comment:
Is there a reason to hard code this? We've historically used `var.region` elsewhere